runtime/dart_isolate.cc

```diff
@@ -445,8 +445,13 @@ void DartIsolate::SetMessageHandlingTaskRunner(
 
   message_handling_task_runner_ = runner;
 
-  message_handler().Initialize(
-      [runner](std::function<void()> task) { runner->PostTask(task); });
+  message_handler().Initialize([runner](std::function<void()> task) {
+    TRACE_EVENT0("flutter", "DartIsolate::PostMessage");
```

Comment:
The PostMessage call is only adding a trace around scheduling a task on an event loop. Is that really necessary? It should be a really tiny slice.

Reply:
I can make it an instance. I spent a while trying to figure out where dart code was being serviced by the engine, this was the entry point for the micro tasks. Maybe it isn't required but I wanted to leave it for future explorers.